src/Breadcrumb/_BreadcrumbItem.js

```diff
@@ -10,11 +10,17 @@ const BreadcrumbItem = ({ url, name, className, children, cssNamespace, ...props
     const renderLink = () => {
         if (!children && url) {
             return (
-                <a className={classnames(`${cssNamespace}-breadcrumb__link`)} href={url}>{name}</a>
+                <a
+                    className={classnames(`${cssNamespace}-breadcrumb__link`)}
+                    href={url}
+                    tabIndex='0'>
```

Comment:
Is there a reason we need to explicitly set a `tabindex` for an `a`?

Reply:
Removing it seems to still work, but I think it's safest to keep it in case there are some browser/screen reader combos that might not tab to these links.